server/routes/gcp/auth.py

```
@@ -224,6 +224,201 @@ def force_disconnect_gcp(user_id):
+            (user_id, org_id),
+        )
+        result = cursor.fetchone()
+        cursor.close()
+        conn.close()
+
+        if result and result[0]:
+            clear_secret_cache(result[0])
+            logging.info(
+                "GCP SA connect: cleared Redis cache for user %s after SA upload", user_id,
+            )
+    except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)